src/database/migrations/0010-insert-canonical_fields.sql

```diff
@@ -0,0 +1,283 @@
+INSERT INTO canonical_fields (id, label, short_code, data_type, created_at) OVERRIDING SYSTEM VALUE VALUES
+(1, 'Organization Name', 'organizationName', '{ type: "string" }', '2022-12-05 11:03:49'),
```

Comment:
Do we actually need to be setting timestamps manually?

```suggestion
(1, 'Organization Name', 'organizationName', '{ type: "string" }', NOW()),
```

Reply:
Technically not. The reason for doing it here is so that everybody who runs this migration will have exactly the same first N rows in the `canonical_fields` table. The timestamp gets generated automatically in the source spreadsheet when adding a label. In the migration sense, it's manual. In the spreadsheet sense, it's somewhat automatic.

Reply:
In other words, this timestamp would represent when the field was added by who edited the sheet. But I am not totally tied to this approach either and it would be fine to keep it consistent with the other tables.

Reply:
(Sorry, just now noticing you responded!)

👍 If we're hoping to maintain data consistency between migrations, that's a good enough reason! If, in practice, we won't make use of that and/or other records will be added in ways that throw a wrench in this -- i.e. if we can't guarantee timestamp consistency always -- then may as well do away with that implicit commitment here by doing `NOW()`. I'll leave that up to you thought, just wanted to flag it!

Reply:
The newer script at https://github.com/PhilanthropyDataCommons/data-scripts/blob/ba13a1d4b92319cc72f0d96e1c5fa888bb56088d/src/generateCanonicalFieldsInserts.ts#L41 based on the newer spreadsheet format lets the created date be generated on `insert` instead.